python/lsst/ts/standardscripts/auxtel/atcam_take_image.py

```diff
@@ -47,7 +47,7 @@ class ATCamTakeImage(salobj.BaseScript):
     def __init__(self, index):
         super().__init__(index=index, descr="Test ATCamTakeImage")
```

Comment:
Existing code, but...does this really "Test" anything? If not could you please improve the description? Naively I would expect it to say something like "Take images with ATCamera"